src/sagas/auth.test.js

```diff
@@ -0,0 +1,60 @@
+import 'jsdom-global/register';
```

Comment:
It's not clear to me why we need this. I mean, I believe the wisdom of @monksp-buildit, but it seems like we should be able to just run these tests without pulling in a DOM.